### DISCUSSION TITLE
preprocess-hourly.ipynb vs preprocess-parquet.ipynb

### DISCUSSION THREAD

**Discussion by selipot** [Cloud-Drift/clouddrift#6] [General]
@philippemiron in both `preprocess-hourly.ipynb` and `preprocess-parquet.ipynb` you generate parquet files, what's the difference? in teh first one is ragged array to parquet and the other akward array to parquet?

**Comment by philippemiron** [comment]
- `preprocess-hourly.ipynb`: creates the netCDF ragged array and two parquets files. Those are unidimensional, so one file contains the `vars['traj']` and the others the `vars['obs']`. Those two parquets files can be opened in pandas.
- `preprocess-parquet.ipynb.ipynb`: reads in the netCDF and transforms into a single parquet file that can be open in awkward array.

Hopefully, all this could be executed in *pargeo-forge*.
